infra/modules/providers/azure/container-registry/tests/integration/acr.go

```diff
@@ -0,0 +1,74 @@
+//  Copyright © Microsoft Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/microsoft/cobalt/test-harness/infratests"
+	"github.com/microsoft/cobalt/test-harness/terratest-extensions/modules/azure"
+	"github.com/stretchr/testify/require"
+)
+
+// healthCheck - Asserts that the deployment was successful.
+func healthCheck(t *testing.T, provisionState string) {
+	require.Equal(t, "Succeeded", provisionState, "The deployment hasn't succeeded.")
+}
+
+// validateDeployment - Asserts that ACR deployment was successful
+func validateDeployment(
```

Comment:
I'd prefer to separate the unit tests (assertion) away from the integration test to follow the same structure in all other modules.

Reply:
I agree with removing the `resource group name` assertion if it's resolvable during the terraform plan. Is it?

Reply:
I'm referring to the assertion on line 44. I see that `resource_group_name` is still needed for the azure sdk.

Reply:
Edit: Assertion is on line 30.